### ISSUE TITLE
Combat Bug?

### ISSUE THREAD

**Issue by RedPrideGames** [dalaranwow/dalaran-wow#5276]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Relogged, restarted and cleared cache
2. Tried a different character/class
3. Tried on other creatures only the wolves in Northshire do this so far

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

[https://www.youtube.com/watch?v=Yn0KP7k8_uc&feature=youtu.be](url)

Jus pointing out some things

Problem? So far, only on wolves has this happened. Yards away and Auto Attack is on
If you cant watch the video I'll explain, I am at a 5 yard range away from wolves and they attack. Its nothing serious jus figured Id point out this bug. The wolf will attack even if you aren't in range. 

Also having problems with abilities like seal of righteous trying to tick after the enemy is already dead

![wowscrnshot_012818_003405](https://user-images.githubusercontent.com/35864198/35479354-3443bd1a-03c3-11e8-88c9-21ad9b7aa130.jpg)

Please solve and close, thanks

**Comment by Rushor** [comment]
Duplicate of https://github.com/dalaranwow/dalaran-wow/issues/1312 and https://github.com/dalaranwow/dalaran-wow/issues/1896

**Comment by RedPrideGames** [comment]
Its not a duplicate...this issue has to do with the Human starting zone...all those other bugs are from different zones

**Comment by RedPrideGames** [comment]
On top of that, those issues are still open and have been for years. No person(s) is going to flip through the timeline of bugs unless its reposted again.

**Comment by RedPrideGames** [comment]
Please reopen or I will post another bug
